csrc/options.h

```diff
@@ -21,6 +21,7 @@ namespace nvfuser {
 //! These can be set through the `NVFUSER_DUMP` environment variable
 //!
 enum class DebugDumpOption {
+  CallStack, //!< Dump the call stack of selected internal function
```

Comment:
I'd also add code comments on how to use this, e.g., DEBUG_PRINT_SCOPE optionally paired with RECORD_AND_RETURN. If you already wrote it down elsewhere in the code, ignore this comment!

Reply:
Added